Packs/AzureWAF/Integrations/AzureWAF/AzureWAF.yml

```diff
@@ -19,6 +19,37 @@ configuration:
   name: resource_group_name
   required: true
   type: 0
+- additionalinfo: Type of authentication - could be Authorization Code Flow (recommended) or Device Code Flow.
+  display: Authentication Type
+  name: auth_type
+  required: false
```

Comment:
Lets make it true (in all other PRs too) with a default vale
@JudahSchwartz means we will force merge